Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -5309,10 +5309,7 @@ class WebCore::RealtimeMediaSourceSupportedConstraints {
     bool supportsBackgroundBlur();
 };
 
-[Nested] class WebCore::RealtimeMediaSource::Type : uint8_t {
-    Audio,
-    Video,
```

Comment:
Why are the values removed? I don't think using a `bool` enum for this is correct.

Reply:
I see in WebCore it is `enum class Type : bool { Audio, Video };` .. I'm not sure what issue you're trying to fix, adding the `enum` and keeping the values isn't OK?

Reply:
The script Source/WebKit/Scripts/generate-serializers.py has assertion (`assert underlying_type != 'bool`') to ensure values are not specified for underlying type bool. I searched for `: bool;` in this file and found the existing use cases. The patch makes Type match those, instead of using a wrong underlying type.

Reply:
Thanks for the explanation @szewai